README.md

```diff
@@ -1,23 +1,23 @@
-# OCAPI Client
+# CommerceCloud OCAPI Client
 
 ```
-__________________________________________________________________
-
-                              ,             /   ,                 
-----__----__----__------__------------__---/--------__----__--_/_-
-  /   ) /   ' /   )   /   ) /       /   ' /   /   /___) /   ) /   
-_(___/_(___ _(___(___/___/_/_______(___ _/___/___(___ _/___/_(_ __
-                    /                                             
-                   /                                              
+_______________________________________________________________________________________________________________________________________________________
+
+                                                            /                    /                                 ,             /   ,                 
+----__----__---_--_---_--_----__---)__----__----__----__---/----__-----------__-/--------__----__----__------__------------__---/--------__----__--_/_-
+  /   ' /   ) / /  ) / /  ) /___) /   ) /   ' /___) /   ' /   /   ) /   /  /   /       /   ) /   ' /   )   /   ) /       /   ' /   /   /___) /   ) /   
+_(___ _(___/_/_/__/_/_/__/_(___ _/_____(___ _(___ _(___ _/___(___/_(___(__(___/_______(___/_(___ _(___(___/___/_/_______(___ _/___/___(___ _/___/_(_ __
+                                                                                                         /                                             
+                                                                                                        /                                              
 ```
 
-[![NPM](https://nodei.co/npm/ocapi-client.png?downloads=true&stars=true)](https://nodei.co/npm/ocapi-client/)
+[![NPM](https://nodei.co/npm/commercecloud-ocapi-client.png?downloads=true&stars=true)](https://nodei.co/npm/commercecloud-ocapi-client/)
 
-[![CircleCI](https://circleci.com/gh/mobify/ocapi-client.svg?style=svg)](https://circleci.com/gh/mobify/ocapi-client)
+[![CircleCI](https://circleci.com/gh/mobify/commercecloud-ocapi-client.svg?style=svg)](https://circleci.com/gh/mobify/commercecloud-ocapi-client)
 
 ## Introduction
 
-ShopApi - JavaScript client for Salesforce OCAPI shop api.
+ShopApi - JavaScript client for CommerceCloud OCAPI shop api.
```

Comment:
I'd still keep "Salesforce" in there... it's a mouthful but the full name is "Salesforce Commerce Cloud"

![image](https://user-images.githubusercontent.com/77138/32397187-cbe60332-c0a5-11e7-83dc-1bd1f15d5d08.png)

Reply:
👍